pegasus/sites.v3/code.org/views/homepage_hero.haml

```diff
@@ -60,7 +60,7 @@
       .col-66.desktop-feature{style: "position:relative;"}
         - heroes_arranged.each_with_index do |entry, index|
           - unless entry[:text].nil_or_empty? || show_census_layout
-            - text = I18n.t(entry[:text])
+            - text = "#{I18n.t(entry[:text])}"
```

Comment:
why is this necessary?

Reply:
The string gets modified a couple lines down, and it started throwing errors because it was trying to modify a frozen string. I still don't understand why it changed from not frozen to frozen, but it doesn't seem ideal to be modifying a local

Looking at it again, I should probably have just used `.dup` instead of this string interpolation.